ledger/eval.go

```diff
@@ -520,8 +521,8 @@ func (eval *BlockEvaluator) transaction(txn transactions.SignedTxn, ad transacti
 		}
 
 		// Transaction already in the ledger?
-		txid := txn.ID()
-		dup, err := cow.isDup(txn.Txn.First(), txn.Txn.Last(), txid, txlease{sender: txn.Txn.Sender, lease: txn.Txn.Lease})
+		cachedTxid = txn.ID()
```

Comment:
nit: use the `cachedTxid` in `return TransactionInLedgerError{txn.ID()}`

Reply:
done!